python/neutron-understack/neutron_understack/nautobot.py

```diff
@@ -98,12 +100,14 @@ def make_api_request(
     def ucvni_create(
         self,
         network_id: str,
+        project_id: str,
         ucvni_group: str,
         network_name: str,
         segment_id: int | None = None,
     ):
         payload = {
             "id": network_id,
+            "tenant": str(uuid.UUID(project_id)),
```

Comment:
So this is validating and I agree with the types but it should probably be on the helper method instead of taking a str.